### ISSUE TITLE
Votes not registering from server back to game (VotingPlugin)

### ISSUE THREAD

**Issue by smmmadden** [NuVotifier/NuVotifier#26]
Hi - I'm trying to get version 2.3.5 to work with my BungeeCord/Spigot 1.12 servers and while the configuration loads fine, commands are accessible in-game, testing the connection from the https://www.craftservers.net/tools/votifier-tester/ site keeps returning "We were unable to talk to your Votifier server." 

Firewall on minecraft servers is open for the ports, the query.port for the server.properties matches the port of NuVotifier and my router is making the port accessible to the public. So I'm wondering if NuVotifier needs to be updated for 1.12 release since it came out after 2.3.5 did?  Nothing on the Spigot page states which version(s) this supports either.

When I tested the server, I tried both my server name, external IP, internal IP and with/without the server startup port and using the query.port value with the token created in the config.yml file.

Thanks,
Steve

**Comment by smmmadden** [comment]
I can test from craftservers.net the server and it finds the server fine.  When I try the votifier test, it is unable to connect. Tested with Telnet and I can access the port.
[INFO] NuVotifier] Votifier enabled on socket /192.168.0.151:8205

Any help would be appreciated.  I'm sure it is something simple, but until I can get this part working, can't setup the voting.

**Comment by smmmadden** [comment]
Update:  I'm using VotingPlugin 5.9.4 and configuration there shows the url to click on and vote to.  What doesn't appear to be happening is registering the vote on minecraft-mp.com back to each server the vote is cast from.  When the player is on any server/world and uses the /vote command, the GUI appears showing the siteurls I can vote to, and then the url appears in the chat space.  Clicking on it takes the player to the the voting server and cast votes appear fine there.  The return back to the game however is not updated.  So the only way to check the votes is going to the voting server which defeats the purpose of having the VotingPlugin to have totals shown. :-(

**Comment by Ichbinjoe** [comment]
@smmmadden  I am going to guess you are running a homehosted server due to the private range ip address. My best guess is that you are not forwarding your port. Try using telnet when not on your home network - just because you can telnet on your local network, doesn't mean that you can on your public ip as well.

**Comment by smmmadden** [comment]
I did work out the issue on this and it wasn't a problem with my hosted servers. It was with the voting servers that were being setup.  For what ever reason that they didn't disclose, they had to setup the server account because of how they wanted me to add something to the server.properties file to verify/validate the connection, and since the servers are in offline mode (Using BungeeCord), their process wouldn't work so we had to do a manual authentication of the servers on their end to get around it.  Drove me nuts since I knew my side was working.  Go figure! :-)
